src/translations/da_DK.json

```diff
@@ -132,28 +132,28 @@
   "CONNECT_PAYMENT_DIRECT_ERROR_EXPIRED_TOKEN": "Dette link er ikke længere gyldigt. Du kan bede Hedvig om at få et nyt.",
   "CONNECT_PAYMENT_DIRECT_ERROR_UNKNOWN": "Åh nej! Der opstod en ukendt fejl.",
   "FOREVER_LANDINGPAGE_TITLE": "Hedvig Forever – sænk din månedlige pris",
-  "FOREVER_INTRO_PAGE_1": "{{REFERRER}}s venner er vores venner.",
+  "FOREVER_INTRO_PAGE_1": "{REFERRER}s venner er vores venner.",
   "FOREVER_INTRO_PAGE_2": "Dette er en invitation til Hedvig Forever.",
   "FOREVER_INTRO_PAGE_3": "For hver ven du inviterer, sænkes din månedlige pris.",
-  "FOREVER_INTRO_PAGE_4": "Lige indtil du når nul og har gratis familieforsikring resten af livet.",
+  "FOREVER_INTRO_PAGE_4": "Lige indtil du når nul og har gratis forsikring resten af livet.",
   "SKIP": "Spring over",
   "FOREVER_INTRO_READY_CTA": "Ja, giv mig et pristilbud",
   "FOREVER_LANDINGPAGE_INPUT_TEXT": "Indtast koden",
   "FOREVER_LANDINGPAGE_BTN_LABEL": "Næste",
   "FOREVER_LANDINGPAGE_INFO_TEXT": "Dette er en eksklusiv hjemmeside, som kun er for inviterede personer. Hvis du ikke har en kode på nuværende tidspunkt, bliver du stillet videre til [vores almindelige hjemmeside](https:\/\/www.hedvig.com\/se\/new-member).",
-  "FOREVER_LANDINGPAGE_DESCRIPTION": "Tillykke! Du er blevet inviteret til Hedvig Forever. Brug min kode {{CODE}} til at sænke din månedlige pris. ",
-  "FOREVER_DOWNLOAD_HEADING": "Velkommen {{FIRST_NAME}}!",
+  "FOREVER_LANDINGPAGE_DESCRIPTION": "Tillykke! Du er blevet inviteret til Hedvig Forever. Brug min kode {CODE} til at sænke din månedlige pris. ",
+  "FOREVER_DOWNLOAD_HEADING": "Velkommen {first_name}!",
```

Comment:
is this intentional?  haven't checked it i just assumed all the replacements are either camel case or screaming snake case

Reply:
Actually i'm assuming this doesn't work. https://github.com/HedvigInsurance/web-onboarding/blob/3dc20f6c63581ce8e87e71b5b56fb9957bce6a07/src/client/pages/Download/sections/DownloadApp.tsx#L185

Reply:
It might be a problem that people adding text keys that aren't us have no way of checking if the keys work or not right? I  mean, I assume the keys _not_ related to the checkout ssn was added by someone else at some other time.

Reply:
Nope, I scanned through them quickly and missed this one. Yeah it might be good to look a bit at the process there. It's a bit of a problem if there's lying around to much update translations. Or I could have just commited my translations

Reply:
Oh, I forgot to say, I updated the key in Lokalise but maybe you saw that already?